src/Microsoft.TestPlatform.Common/Telemetry/TelemetryDataConstants.cs

```diff
@@ -117,6 +115,19 @@ public static class TelemetryDataConstants
 
     public static string AttachmentsProcessingState = "VS.AttachmentsProcessing.State";
 
+    // *********************Test Sessions****************************
+    public static string ParallelEnabledDuringStartTestSession = "VS.TestSession.ParallelEnabled";
```

Comment:
I know you only copy/pasted pattern in place but could we make the new added "constants" readonly?

```suggestion
    public static readonly string ParallelEnabledDuringStartTestSession = "VS.TestSession.ParallelEnabled";
```

Reply:
I think we can update others as well...looks safe

Reply:
It will be a change to the public API (part of the file) so although it feels safe it's probably something we should agree upon and do as a separate PR.

Reply:
I agree the change is needed but I also agree that a separate PR would probably be best. I'd like to keep the changes in this PR as relevant as possible to ease the review process.

Reply:
oh we're shipping also this stuff public 😢 never mind.

Reply:
Could we still make the newly added constants to be `readonly`? We don't know when/if we will be able to change the others so let's try to make sure the new ones cannot be altered.

Reply:
it's ok...very confusing to dev...every time one failed build for newcomers 😄